meson.build

```
@@ -51,6 +52,15 @@ cdata.set('DEFAULT_VIDEO_FRAMERATE', 15)
 cdata.set('DEFAULT_VIDEO_BITRATE', 1000000)
 cdata.set('DEFAULT_VIDEO_CODEC', 'GAEGULI_VIDEO_CODEC_H264_X264')
 
+cdata.set_quoted('GAEGULI_PIPELINE_WORKER',
+  join_paths(gaeguli_bindir, 'gaeguli_pipeline_worker'))
+cdata.set_quoted('GAEGULI_PIPELINE_WORKER_ALT',
+  join_paths(meson.build_root(), 'gaeguli', 'workers', 'pipeline_worker','gaeguli_pipeline_worker'))
+cdata.set_quoted('GAEGULI_TARGET_WORKER',
+  join_paths(gaeguli_bindir, 'gaeguli_target_worker'))
+cdata.set_quoted('GAEGULI_TARGET_WORKER_ALT',
+  join_paths(meson.build_root(), 'gaeguli', 'workers', 'target_worker','gaeguli_target_worker'))
```

The paths are not correct

```suggestion
cdata.set_quoted('GAEGULI_PIPELINE_WORKER_ALT',
  join_paths(meson.build_root(), 'subprojects', 'gaeguli', 'gaeguli', 'workers', 'pipeline_worker', 'gaeguli_pipeline_worker'))
cdata.set_quoted('GAEGULI_TARGET_WORKER',
  join_paths(gaeguli_bindir, 'gaeguli_target_worker'))
cdata.set_quoted('GAEGULI_TARGET_WORKER_ALT',
  join_paths(meson.build_root(), 'subprojects', 'gaeguli', 'gaeguli', 'workers', 'target_worker', 'gaeguli_target_worker'))
```